bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml

```diff
@@ -56,6 +56,12 @@ spec:
       hostNetwork: true
       hostPID: true
       priorityClassName: "system-node-critical"
+      {{if .ENABLE_OVN_NODE_PROXY}}
```

Comment:
A comment here explaining what this does would be good :)

Reply:
Done